packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -40,6 +41,13 @@ interface Props {
 	 * `storageAddOmns` TODO: should become a required prop.
 	 */
 	storageAddOns: ( AddOnMeta | null )[] | null;
+
+	/**
+	 * Whether to include discounts from plan proration.
+	 * This is applicable only if a siteId is passed to this hook.
+	 * If true, the pricing includes discounts from upgrade credits.
+	 */
+	withProratedDiscounts?: boolean;
```

Comment:
NIT - I think "prorated discount" may not be correct grammatically. Meaning, using "prorated" adjective e.g. can a discount be prorated? 

"prorated price" sounds correct, "pro rata discount", "discount from proration"

Obviously I could be wrong.

Reply:
Good point, thanks for raising this! `withDiscountFromProration` is slightly verbose, but I think it explains the arg correctly.

Reply:
As you like! Indeed `withDiscountFromProration` may be slightly verbose. 
I'm also happy with the original `withProratedDiscounts` irrespective tbh. "prorate" is probably more common term in general to have in variables